hw/ip/clkmgr/doc/theory_of_operation.md

```diff
@@ -182,6 +188,9 @@ In cases where there is a frequency mismatch, the clock manager supports clock d
 The divided frequency is not assumed to be synchronous with its source and are thus treated like another asynchronous branch.
 Further, the clock dividers are hardwired and have no software control, this is to further ensure there are no simple paths for faulty or malicious software to tamper.
 
+Clock division is mainly used as part of the [external clock switch support](#external-clock-switch-support), since the external clock may be slower than the internal clock source.
```

Comment:
```suggestion
Clock division can be modified for [external clock switch support](#external-clock-switch-support), since the external clock may be slower than the internal clock source.
```

Reply:
done